### PR TITLE
Blogstyling

### PR DESCRIPTION
**Pull request by kellynauert**

### CODE REVIEW

File: src/components/QuestionRender.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  },
+});
+
+function QuestionRender({
+  answer, required, questionID, isReadOnly, editMode,
+  toggleEditMode, getClass, getRequiredClass, blogQuestions,
+}) {
+  const classes = useStyles();
+
+  function onChangeHandler(e) {
```
Comment (cabarnes) [inline src/components/QuestionRender.js:52]: Can this be changed to an arrow function to be consistent with the rest of the application?

File: src/components/Blog.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  const getRequiredClass = (required, answer) => {
+    if (required && !answer && !isReadOnly) {
+      return classes.unfinishedRequired;
+    }
+    return null;
+  };
+
+  return (
+    <Grid container direction="column">
+      {questions.length === 0 && isReadOnly ? (
```
Comment (cabarnes) [inline src/components/Blog.js:83]: Could all of these where `{` is on the same line as the javascript be formatted like below to be consistent with the rest of the application?
```
{
  <javascript> ? (
  ) : (
  )
}
```
Basically, have the `{` and `}` on their own lines with the rest indented within it

File: src/components/Blog.js [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+          <Typography
+            variant="caption"
+            style={{ color: 'grey', fontStyle: 'italic' }}
+          >
+            No Blog Post Available
+          </Typography>
+        ) : (
+          <>
+            {
+                !isReadOnly ? (
```
Comment (cabarnes) [inline src/components/Blog.js:94]: These seem to have indented 4 spaces

Comment (kellynauert) [reply src/components/Blog.js]: I think I fixed them all now. That is how the ESLint formatter was formatting them for some reason.

**Review by cabarnes — APPROVED** [review]
Great job! This looks really nice

**Pull request merged**